src/app/clusters/localization-configuration-server/BUILD.gn

```diff
@@ -11,5 +11,22 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-group("localization-configuration-server") {
+
+import("//build_overrides/build.gni")
+import("//build_overrides/chip.gni")
+
+source_set("localization-configuration-server") {
+  sources = [
+    "localization-configuration-cluster.cpp",
+    "localization-configuration-cluster.h",
+    "localization-configuration-logic.cpp",
+    "localization-configuration-logic.h",
```

Comment:
While split between cluster and logic is generaly a good idea, historically we have found that this may increase flash more. 

I am looking at the changes now:
   - we have 100 bytes increase in several platforms (probably ok)
   - some platforms increase by 600-800 bytes (psoc6, telink, nxp lock, esp32 all-clusters)
   
 Do we know why? Should we maybe just have clusters and not split logic?